addons/find-bar/blockly/Utils.js

```diff
@@ -18,11 +18,6 @@ export default class Utils {
     // this._myFlash = { block: null, timerID: null, colour: null };
     this.offsetX = 32;
     this.offsetY = 32;
-    this.navigationHistory = new NavigationHistory();
```

Comment:
I think it makes sense to keep `NavigationHistory` a separate class - it acts as a namespace for the methods related to navigation. You can add a constructor to it that takes `addon` as a parameter.